ml4kp_bridge/SpacePoint control
std_msgs/Duration duration